test/f16-vmulcaddc-minmax.cc

```diff
@@ -4,629 +4,28 @@
-          .Test(xnn_f16_vmulcaddc_minmax_ukernel_c16__fma3_2x, xnn_init_f16_minmax_scalar_params);
-      }
-    }
-  }
-
-  TEST(F16_VMULCADDC_MINMAX_C16__FMA3_2X, rows_gt_2) {
-    TEST_REQUIRES_X86_FMA3;
-    for (size_t rows = 3; rows < 4; rows++) {
-      for (size_t channels = 1; channels <= 80; channels += 15) {
-        VMulCAddCMicrokernelTester()
-          .channel_tile(16)
-          .channels(channels)
-          .rows(rows)
-          .Test(xnn_f16_vmulcaddc_minmax_ukernel_c16__fma3_2x, xnn_init_f16_minmax_scalar_params);
-      }
-    }
-  }
-
-  TEST(F16_VMULCADDC_MINMAX_C16__FMA3_2X, input_stride) {
-    TEST_REQUIRES_X86_FMA3;
-    for (size_t rows = 1; rows <= 6; rows += 1) {
-      for (size_t channels = 1; channels <= 80; channels += 15) {
-        VMulCAddCMicrokernelTester()
-          .channel_tile(16)
-          .channels(channels)
-          .rows(rows)
-          .input_stride(83)
-          .Test(xnn_f16_vmulcaddc_minmax_ukernel_c16__fma3_2x, xnn_init_f16_minmax_scalar_params);
-      }
-    }
-  }
-
-  TEST(F16_VMULCADDC_MINMAX_C16__FMA3_2X, output_stride) {
-    TEST_REQUIRES_X86_FMA3;
-    for (size_t rows = 1; rows <= 6; rows += 1) {
-      for (size_t channels = 1; channels <= 80; channels += 15) {
-        VMulCAddCMicrokernelTester()
-          .channel_tile(16)
-          .channels(channels)
-          .rows(rows)
-          .output_stride(83)
-          .Test(xnn_f16_vmulcaddc_minmax_ukernel_c16__fma3_2x, xnn_init_f16_minmax_scalar_params);
-      }
-    }
-  }
-
-  TEST(F16_VMULCADDC_MINMAX_C16__FMA3_2X, inplace) {
-    TEST_REQUIRES_X86_FMA3;
-    for (size_t rows = 1; rows <= 6; rows += 1) {
-      for (size_t channels = 1; channels <= 80; channels += 15) {
-        VMulCAddCMicrokernelTester()
-          .channel_tile(16)
-          .channels(channels)
-          .rows(rows)
-          .inplace(true)
-          .Test(xnn_f16_vmulcaddc_minmax_ukernel_c16__fma3_2x, xnn_init_f16_minmax_scalar_params);
-      }
-    }
-  }
-
-  TEST(F16_VMULCADDC_MINMAX_C16__FMA3_2X, qmin) {
-    TEST_REQUIRES_X86_FMA3;
-    for (size_t rows = 1; rows <= 6; rows += 1) {
-      for (size_t channels = 1; channels <= 80; channels += 15) {
-        VMulCAddCMicrokernelTester()
-          .channel_tile(16)
-          .channels(channels)
-          .rows(rows)
-          .qmin(128)
-          .Test(xnn_f16_vmulcaddc_minmax_ukernel_c16__fma3_2x, xnn_init_f16_minmax_scalar_params);
-      }
-    }
-  }
-
-  TEST(F16_VMULCADDC_MINMAX_C16__FMA3_2X, qmax) {
-    TEST_REQUIRES_X86_FMA3;
-    for (size_t rows = 1; rows <= 6; rows += 1) {
-      for (size_t channels = 1; channels <= 80; channels += 15) {
-        VMulCAddCMicrokernelTester()
-          .channel_tile(16)
-          .channels(channels)
-          .rows(rows)
-          .qmax(128)
-          .Test(xnn_f16_vmulcaddc_minmax_ukernel_c16__fma3_2x, xnn_init_f16_minmax_scalar_params);
-      }
-    }
-  }
-#endif  // XNN_ARCH_X86 || XNN_ARCH_X86_64
+#include "next_prime.h"
```

Comment:
If this is really necessary, it should be in vmulcaddc-microkernel-tester.h. But I don't think it is.

Reply:
Didn't notice , Now it's been removed